src/mount.ts

```diff
@@ -0,0 +1,217 @@
+const completionSpec: Fig.Spec = {
+  name: "mount",
+  description: "Mount disks and manage subtrees",
+  args: [
+    {
+      name: "Disk/loopfile",
+      template: "filepaths",
+      generators: [
+        {
+          script: "cat /proc/partitions", // this way we don't depend on lsblk
+          postProcess: (out) => {
+            return out
+              .split("\n")
+              .splice(2, out.length)
+              .map((line) => "/dev/" + line.split(" ").pop())
+              .filter((x) => x != "/dev/")
+              .map((blk) => {
+                return { name: blk, description: "Block device" };
+              });
+          },
+        },
+        {
+          script: "ls /dev/mapper", // usually LUKS encrypted partitions are here
```

Comment:
Guarantees using system ls instead of bash builtin
```suggestion
          script: "\\command ls -1 /dev/mapper", // usually LUKS encrypted partitions are here
```

Reply:
Unsure about this one. I'd prefer we tried to keep script blocks as vanilla as possible. At least under the hood we are not even running aliases or the rc files at all.

Reply:
Ah I see. Avoiding the bash builtin, maybe we should rethink this API.

Reply:
is `-1` part of POSIX ls? GNU has added a lot of stuff and it's difficult to make things portable when using coreutils in scripts.

Reply:
Not sure if it's posix (pretty sure it is) but `-1` is definitely on gnu ls

Reply:
Yup I'm like 99% sure `-1` is a posix thing